ansible-chatbot-run.yaml

```diff
@@ -0,0 +1,133 @@
+    config:
+      kvstore:
+        type: sqlite
+        namespace: null
+        db_path: ${env.SQLITE_STORE_DIR:/.llama/distributions/ansible-chatbot}/aap_faiss_store.db
+  safety:
+  - provider_id: llama-guard
+    provider_type: inline::llama-guard
+    config:
+      excluded_categories: []
+  - provider_id: lightspeed_question_validity
+    provider_type: inline::lightspeed_question_validity
+    config:
+      model_id: ${env.INFERENCE_MODEL}
+      model_prompt: |-
+        Instructions:
+        - You are a question classifying tool
+        - You are an expert in ansible
+        - Your job is to determine where or a user's question is related to ansible technologies and to provide a one-word response
+        - If a question appears to be related to ansible technologies, answer with the word ${allowed}, otherwise answer with the word ${rejected}
+        - Do not explain your answer, just provide the one-word response
+        
+        
+        Example Question:
+        Why is the sky blue?
+        Example Response:
+        ${rejected}
+        
+        Example Question:
+        Can you help generate an ansible playbook to install an ansible collection?
+        Example Response:
+        ${allowed}
+        
+        Example Question:
+        Can you help write an ansible role to install an ansible collection?
+        Example Response:
+        ${allowed}
+        
+        Question:
+        ${message}
+        Response:
+      invalid_question_response: |-
+        Hi, I'm the Ansible Lightspeed Intelligent Assistant, I can help you with questions about Ansible, 
+        please ask me a question related to Ansible.
+  agents:
+  - provider_id: meta-reference
+    provider_type: inline::meta-reference
+    config:
+      persistence_store:
+        type: sqlite
+        namespace: null
+        db_path: ${env.SQLITE_STORE_DIR:/.llama/distributions/ansible-chatbot}/agents_store.db
+  datasetio:
+  - provider_id: localfs
+    provider_type: inline::localfs
+    config:
+      kvstore:
+        type: sqlite
+        namespace: null
+        db_path: ${env.SQLITE_STORE_DIR:/.llama/distributions/ansible-chatbot}/localfs_datasetio.db
+  telemetry:
+  - provider_id: meta-reference
+    provider_type: inline::meta-reference
+    config:
+      service_name: ${env.OTEL_SERVICE_NAME:}
+      sinks: ${env.TELEMETRY_SINKS:console,sqlite}
+      sqlite_db_path: ${env.SQLITE_STORE_DIR:/.llama/distributions/ansible-chatbot}/trace_store.db
+  tool_runtime:
+  - provider_id: rag-runtime
+    provider_type: inline::rag-runtime
+    config: {}
+  - provider_id: lightspeed
+    provider_type: remote::lightspeed
+    config: {}
+metadata_store: null
+models:
+- metadata: {}
+  model_id: ${env.INFERENCE_MODEL}
+  provider_id: rhosai_vllm_dev
+  provider_model_id: null
+- metadata:
+    embedding_dimension: 384
+  model_id: all-MiniLM-L6-v2
+  provider_id: inline_sentence-transformer
+  model_type: embedding
+shields:
```

Comment:
Can you please comment out/remove the `shield`?

@TamiTakamiya advised we don't use `road-core`'s Question Validation in `ansible-chatbot-service` and so we don't really need it with `llama-stack` either. You could remove the related `safety` provider declaration too; but removing this block will mean it is unavailable to `Agent`'s.